.github/workflows/test.yml

```diff
@@ -269,16 +276,58 @@ jobs:
   test-spell-check:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+
+      - uses: actions/checkout@v3
         with:
-          repository: FreeRTOS/coreMQTT
-          ref: main
+          repository: skptak/coreMQTT
+          ref: CI-CD-Updates
           path: coreMQTT
+
       - name: Test spell check action
         uses: ./spellings
         with:
           path: coreMQTT
+
+  test-spell-checker-find-mistake:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/checkout@v3
+        with:
+          repository: skptak/coreMQTT
```

Comment:
Had totally missed this originally, but same issue as Kody's above.

Reply:
Requires a wordlist and config file that does not exist in current mainline. The PR for coreMQTT using this action can be found [here](https://github.com/FreeRTOS/coreMQTT/pull/260)